pos_gift_ticket/static/src/xml/pos.xml

```diff
@@ -28,6 +26,9 @@
                 <br />
```

Comment:
It would be great to put Phone, User and Shop in html tags (like div) to make it possible to inherit and personalize Gift Ticket content.